Open Judge System/OJS.Common/Extensions/StringExtensions.cs

```diff
@@ -194,5 +194,7 @@ public static string PascalCaseToText(this string input)
 
             return result.ToString();
         }
+
+        public static int CountNewLines(this string input) => Regex.Matches(input, "\n").Count + 1;
```

Comment:
If we use Environment.NewLine it won't work in the case where you read an LF file under windows because the Environment.NewLine will be CRLF. A simple solution to this is to just assume all new line endings contain \n and just lookup for that.